packages/contracts-bedrock/src/L1/OptimismPortal2.sol

```diff
@@ -162,74 +148,98 @@ contract OptimismPortal2 is Initializable, ResourceMetering, ISemver {
     /// @param success        Whether the withdrawal transaction was successful.
     event WithdrawalFinalized(bytes32 indexed withdrawalHash, bool success);
```

Comment:
Assuming we intentionally are leaving these event names unchanged because we watch them with monitoring or something similar?

Reply:
Yep